config/v1/types_node.go

```diff
@@ -48,7 +48,13 @@ type NodeSpec struct {
 	WorkerLatencyProfile WorkerLatencyProfileType `json:"workerLatencyProfile,omitempty"`
 }
 
-type NodeStatus struct{}
+type NodeStatus struct {
+	// Conditions contain the details and the current state of the nodes.config object
+	// +listType=map
+	// +listMapKey=type
```

Comment:
If you check other versions of this field, they have strategic merge patch keys as well, can you add those

Reply:
Addressed